okta/resource_clients/user_schema_client.py

```diff
@@ -67,7 +67,7 @@ async def get_application_user_schema(
 
         try:
             result = UserSchema(
-                self.form_response_body(response.get_body())
+                response.get_body()
```

Comment:
Not formatting the response body before creating the response in order to not change any custom attributes which may come through in here